Examples/NoModules/Chapter 02/Ex2_03A.cpp

```diff
@@ -0,0 +1,31 @@
+// Expressions with mixed variables types
+// (The difference with the original example is the type of fish_length)
+#include <iostream>
+#include <numbers>   // For the pi constant
+#include <cmath>     // For the square root function
+
+int main()
+{
+  // 2 square feet pond surface for every 6 inches of fish
+  const double fish_factor { 2.0/0.5 };  // Area per unit length of fish
+  const double inches_per_foot { 12.0 };
```

Comment:
The text says to replace this double constant with:
 const unsigned int inches_per_foot {12};